support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.tsx

```diff
@@ -1,11 +1,8 @@
 import * as React from 'react';
 // constants
-import DigitalPackshot from 'components/packshots/digital-packshot';
-import DigitalPackshotHero from 'components/packshots/digital-packshot-hero';
 import GuardianWeeklyPackShot from 'components/packshots/guardian-weekly-packshot';
 import GuardianWeeklyPackShotHero from 'components/packshots/guardian-weekly-packshot-hero';
 import PaperPackshot from 'components/packshots/paper-packshot';
-import PremiumAppPackshot from 'components/packshots/premium-app-packshot';
```

Comment:
Also this

Reply:
Think this was already been done as part of second commit? Have I got that right?

Reply:
Sorry yes the files not removed, will do this

Reply:
Removed